scm/src/GFS_typedefs.F90

```diff
@@ -3631,10 +3611,6 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%iau_drymassfixer = iau_drymassfixer
     if(Model%me==0) print *,' model init,iaufhrs=',Model%iaufhrs
 
-    !--- debug flag
```

Comment:
I think that this change should be reverted. Otherwise, Model%debug is used before being set.